src/finufft.cpp

```diff
@@ -67,31 +68,8 @@ Design notes for guru interface implementation:
 */
 
 
-int* gridsize_for_fftw(finufft_plan* p){
```

Comment:
maybe instead of moving I might be able to declare as `static`, can try on another one.... I didn't think about it last night.